src/components/Button/index.examples.js

```diff
@@ -8,14 +8,14 @@ const standardButtonExamples = [
   {type: 'primary', children: 'Primary'},
   {type: 'success', children: 'Success'},
   {type: 'warning', children: 'Warning'},
-  {type: 'danger', children: 'Danger'}
+  {type: 'danger', children: 'Danger'},
+  {type: 'orange', children: 'Orange'},
+  {type: 'turquoise', children: 'turquoise'}
```

Comment:
Not necessary for this PR, but in the future, it would be nice if all the types were either semantic or colors, not mixed as they currently are. I'd prefer if we as a team came up with semantic meanings for the colors and styles used so the `type` could represent that; so maybe `warning` is always orange and large, and `more-info` is always small and white outlined etc. An idea at least :)